content/webapp/utils/iiif/v3/index.ts

```diff
@@ -339,6 +339,31 @@ export function getTokenService(
       )
     : clickThroughService?.service;
 }
+type AuthServices = {
+  active?: TransformedAuthService;
+  external?: TransformedAuthService;
+};
+
+export function getAuthServices({
+  auth,
+  authV2,
+}: {
+  auth?: Auth;
```

Comment:
As we've just found out, auth cannot be undefined (it can be an empty object), so we should change the type and the checks to consider that.